x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/gcp_credentials_form/gcp_credential_form.tsx

```diff
@@ -349,7 +349,7 @@ const useCloudShellUrl = ({
 
 export const getGcpCredentialsType = (
   input: Extract<NewPackagePolicyPostureInput, { type: 'cloudbeat/cis_gcp' }>
-): GcpCredentialsType | undefined => input.streams[0].vars?.setup_access.value;
+): GcpCredentialsType | undefined => input.streams[0].vars?.['gcp.credentials.type'].value;
```

Comment:
there is no need to save `setup_access` value because it can only be either `manual` or `google_cloud_shell`